packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -97,51 +97,59 @@ const PipelineRuns = ({
     }
   });
 
-  const pipelineRunsFormatted = pipelineRuns.map(pipelineRun => ({
-    id: `${pipelineRun.metadata.namespace}:${pipelineRun.metadata.name}`,
-    name: (
-      <Link
-        to={createPipelineRunURL({
-          namespace: pipelineRun.metadata.namespace,
-          pipelineRunName: createPipelineRunDisplayName({
-            pipelineRunMetadata: pipelineRun.metadata
-          })
-        })}
-      >
-        {pipelineRun.metadata.name}
-      </Link>
-    ),
-    pipeline: !hidePipeline && (
-      <Link
-        to={createPipelineRunsByPipelineURL({
-          namespace: pipelineRun.metadata.namespace,
-          pipelineName: pipelineRun.spec.pipelineRef.name
-        })}
-      >
-        {pipelineRun.spec.pipelineRef.name}
-      </Link>
-    ),
-    namespace: !hideNamespace && pipelineRun.metadata.namespace,
-    status: (
-      <div className="definition">
-        <div
-          className="status"
-          data-status={getStatus(pipelineRun).status}
-          data-reason={getStatus(pipelineRun).reason}
-        >
-          <div className="status-icon">
-            {getPipelineRunStatusIcon(pipelineRun)}
+  const pipelineRunsFormatted = pipelineRuns.map(pipelineRun => {
+    const { namespace, annotations } = pipelineRun.metadata;
+    const pipelineRunName = createPipelineRunDisplayName({
+      pipelineRunMetadata: pipelineRun.metadata
+    });
+    const pipelineRefName = pipelineRun.spec.pipelineRef.name;
+    const pipelineRunType = pipelineRun.spec.type;
+    const { reason, status } = getStatus(pipelineRun);
+    const statusIcon = getPipelineRunStatusIcon(pipelineRun);
+    const pipelineRunStatus = getPipelineRunStatus(pipelineRun, intl);
+    const url = createPipelineRunURL({
+      namespace,
+      pipelineRunName,
+      annotations
+    });
+    return {
+      id: `${namespace}:${pipelineRunName}`,
+      name: <Link to={url}>{pipelineRunName}</Link>,
```

Comment:
Noticed another small issue here, we need to make sure we handle `createPipelineRunURL` not returning a value as we do for TaskRuns. Consumers may not have a location to link to in all cases so may want to display just the name without making it a link.

```suggestion
      name: url ? <Link to={url}>{pipelineRunName}</Link> : pipelineRunName,
```